global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.404",
+    "version": "8.0.202",
```

Comment:
Why not to take the latest, 8.0.203?

Reply:
latestFeature should be allowed as an option, otherwise it is crazy to need to update this file to test a new sdk version